### DISCUSSION TITLE
How to get the YOLO

### DISCUSSION THREAD

**Discussion by oscarxdev** [Schweinepriester/github-profile-achievements#349] [Q&A]
How to get the YOLO

**Comment by L4B0MB4** [comment]
yolo!

**Comment by 0xHericles** [comment]
Just have a pull request merged without any reviews.

**Comment by Semanteo** [comment]
Look in the readme there is a thread on it !

**Comment by WHAHA-HA** [comment]
Under your repository name, click Pull requests. In the list of pull requests, click the pull request that you'd like to ask a specific person or a team to review. Navigate to Reviewers in the right sidebar. To request a review from a suggested person under Reviewers, next to their username, click Request.
You can see more info at [requesting-a-pull-request-review](https://docs.github.com/articles/requesting-a-pull-request-review)
PS: It'd be appreciated if you accept my answer. Thank you.
